frontend/test/pytest/test_gradient.py

```diff
@@ -1172,5 +1175,41 @@ def interpreted(x):
     assert np.allclose(compiled(inp), interpreted(inp))
 
 
+class TestGradientErrors:
```

Comment:
The changelog also mentions that zne would raise an error, should add a test for this as well :)

Reply:
https://github.com/PennyLaneAI/catalyst/pull/204/commits/1390d23e18fb798ff762bdcbd41d10ba3fd710d7